pod/perldelta.pod

```diff
@@ -80,16 +322,115 @@ as an updated module in the L</Modules and Pragmata> section.
 
 =head1 Performance Enhancements
 
-XXX Changes which enhance performance without changing behaviour go here.
-There may well be none in a stable release.
+=over 4
 
-[ List each enhancement as an =item entry ]
+=item *
 
-=over 4
+Constant-folded strings are now sharable via the Copy-on-Write mechanism.
+[L<GH #22163|https://github.com/Perl/perl5/pull/22163>]
+
+The following code would previously have allocated eleven string buffers,
+each containing one million "A"s:
+
+C<my @scalars; push @scalars, ("A" x 1_000_000) for 0..9;>
+
+Now a single buffer is allocated and shared between a CONST OP and
+the ten scalar elements of L<@scalars>.
+
+Note that any code using this sort of constant to simulate memory leaks
+(perhaps in test files) must now permute the string in order to trigger
+a string copy and the allocation of separate buffers. For example,
+C<("A" x 1_000_000).time> might be a suitable small change.
 
 =item *
 
-XXX
+C<Perl_leave_adjust_stacks>: additional efficiency for mortal copies.
```

Comment:
I don't think this needs to be included. It's unlikely to be very user-noticeable.